### ISSUE TITLE
A11y_SQLAzureDataStudio_SQLServerBigDataCluster_Summary_Keyboard:Focus is not visible on the copy and close button. 

### ISSUE THREAD

**Issue by fsteffi** [microsoft/azuredatastudio#9321]
[“Check out Accessibility Insights! ](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier.”

#A11y_SQLAzureDataStudioBenchmark;#A11yMAS;#A11ySev2;#A11yTCS;#SQL Azure Data Studio;#Benchmark;#DesktopApp;#MacOs;#MAS2.4.7;#Keyboard;

**Environment Details:**
Application Name: Azure Data Studio
Version: 1.15.0-insider
Commit: 3a8b74a
Date: 2020-02-11T05:49:41.181Z
VS Code: 1.42.0
Electron: 7.1.11
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Darwin x64 19.3.0
Mac OS:10.15.3

**Additional Details:**
Mac OS:10.15.3
MAS References:MAS2.4.7

**Repro Steps:**
1. Launch Azure Data Studio.
2. Select the Connections icon (if not already selected) from left bar.
3. Click on the ellipses '...' and select "New Deployment".
4. Select "SQL Server Big Data Cluster" card under deployment options.
5. Select the version & checkbox.
6. Navigate till Selet button & hit enter.
7. Select Configuration profile --> Update Azure settings --> Update Cluster settings --> Update Service settings & hit Next.
8. Navigate to the copy and close button using tab key.

**Actual:**
On navigating using tab to the close and copy button the focus is not visible on them.

**Expected:**
The focus should be clearly visible through black dotted lines on the button.

**User Impact:**
Keyboard users will face difficulty in knowing where the current focus is.

**Attachment link for Reference :**
[A11y_SQLAzureDataStudio_SQLServerBigDataCluster_Summary_Keyboard1.zip](https://github.com/microsoft/azuredatastudio/files/4322610/A11y_SQLAzureDataStudio_SQLServerBigDataCluster_Summary_Keyboard1.zip)

**Comment by alanrenmsft** [comment]
focus rectangle not visible for information level

**Comment by BhanuPrakashGit** [comment]
As "SQL Server Big Data Cluster" feature has been removed.
**GithubTags:**#Closed;

**Comment by mstechie** [comment]
GitHubTags:#SQLADS-MAC-Feb2020;
